conformance/lang/expressions/xml-attribute-access-expr/xml_attribute_access_expr.balt

```diff
@@ -0,0 +1,371 @@
+    io:println(x.<messages>.ns:id); // @output 10
+    io:println(x/*.<note>[0].ns:id); // @output 11
+
+    x = xml `<messages ns:id1="10" ns:id2="101" ns:id3="102">
+                    <note ns:id1="11">
+                        <to>Tove</to>
+                    </note>
+                    <note ns:id2="12">
+                        <to>Jani</to>
+                    </note>
+                    <note ns:id3="13">
+                        <to>Jan</to>
+                    </note>
+                </messages>`;
+    io:println(x.ns:id3); // @output 102
+
+    x = xml `<root ns:id1='10' ns:id2='11'/>`;
+    io:println(x.ns:id1); // @output 10
+
+    x = xml `<messages>
+                    <note ns:id="11">
+                        <to>Tove</to>
+                    </note>
+                </messages>`;
+    io:println(x/<note>.ns:id); // @output 11
+
+    x = xml `<messages>
+                    <note ns:id="11">
+                        <to>Tove</to>
+                    </note>
+                    <note ns:id="12">
+                        <to>Jane</to>
+                    </note>
+                </messages>`;
+    io:println(x/<note>[0].ns:id); // @output 11
+    io:println(x/<note>[1].ns:id); // @output 12
+
+    x = xml
+        `<items ns:id='9'>
+            <book ns:id='10'>
+                <name>A Study in Scarlet</name>
+                <author ns:id='11'><name>Arthur Conan Doyle<Date ns:id='12'></Date></name></author>
+            </book>
+            <planner ns:id='13'>Daily Planner<kind ns:id='14'>day</kind><pages ns:id='15'>365</pages></planner>
+            <book id='20'>
+                <name>The Sign of Four</name>
+                <author><name>Arthur Conan Doyle</name></author>
+            </book>
+            <pen><kind>marker</kind><color ns:id='21'>blue</color></pen>
+        </items>`;
+    io:println(x.<items>.ns:id); // @output 9
+    io:println(x.<items>/<book>[0].ns:id); // @output 10
+    io:println(x.<items>/<book>[1].id); // @output 20
+    io:println(x.<items>/<book>[0]/<author>.ns:id); // @output 11
+    io:println(x.<items>/<book>[0]/<author>/<name>/<Date>.ns:id); // @output 12
+    io:println(x.<items>/<planner>/<pages>.ns:id); // @output 15
+    io:println(x.<items>/<pen>/<color>.ns:id); // @output 21
+    io:println(x/*.<book>[0].ns:id); // @output 10
+    io:println(x/*.<book>[0]/<author>.ns:id); // @output 11
+    io:println(x/*.<pen>/<color>.ns:id); // @output 21
+    io:println(x/<*>.<book>[1].id); // @output 20
+    io:println(x/<*>.<planner>/<kind>.ns:id); // @output 14
+    io:println(x/<*>.<pen>/<color>.ns:id); // @output 21
+    io:println(x/**/<items>/<book>[0]/<author>/<name>/<Date>.ns:id); // @output 12
+    io:println(x/**/<items>/<book>[1].id); // @output 20
+    io:println(x/**/<items>/<planner>/<kind>.ns:id); // @output 14
+    io:println(x/**/<items>/<pen>/<color>.ns:id); // @output 21
+    io:println(x/**/<items>/<book>[0]/<author>/<name>/<Date>.ns:id); // @output 12
+}
+
```

Comment:
Don't we have error tests for invalid static types?

Reply:
Added more tests.